### ISSUE TITLE
Collect common plant dataset

### ISSUE THREAD

**Issue by mattmotoki** [HACC2018/ohia.ai#10]
Tasks:
* download/scrape images of plants
* collect meta data; e.g., stories, uses, taxonomic data, status (native, invasive, endangered, etc.)
* filter out all non-plant images

Plants:
* hibiscus
* plumeria
* pineapple
* ginger
* anthurium
* heliconia
* bougainvillea

**Comment by windybasket** [comment]
scraped instagram for ~400 pictures in each hashtag. Includes non-plant pictures. Will go through images depending on how many needed for each class. [https://drive.google.com/drive/folders/1TauCtJxGnEOYLX0_iKVXltuS2hNVNf3d?usp=sharing](url)

**Comment by windybasket** [comment]
Deleted misclassified images, ~2600 images total in common_v2

**Comment by windybasket** [comment]
I went through all the pictures manually for V2 to make ground truth certain. The pictures are all from mobile/regular life so the settings are variable ranging from selfies to coffee shop commercials. I used this github to scrape posts with a free AWS (fastest internet) https://github.com/rarcega/instagram-scraper. Only edited to get --maximum 500 posts and command line loop to get 7 hashtags. I'm not sure about the native/invasives, but I'm in the process of testing. The tensorflow retraining uses folders as classes so I don't need to create a csv, but it should be straightforward with ls >
